docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -11,57 +11,61 @@ distributes these calculations across your cluster. You can then feed this
 aggregated data into the {ml-features} instead of raw results, which
 reduces the volume of data that must be considered while detecting anomalies.
 
-TIP: If you use a terms aggregation and the cardinality of a term is high, the
-aggregation might not be effective and you might want to just use the default
-search and scroll behavior.
+TIP: If you use a terms aggregation and the cardinality of a term is high but
+still significantly less than your total number of documents, use composite
+aggregations
+experimental:[support for composite aggregations inside datafeeds is currently experimental].
 
 [discrete]
 [[aggs-limits-dfeeds]]
 == Requirements and limitations
 
-There are some limitations to using aggregations in {dfeeds}. Your aggregation
-must include a `date_histogram` aggregation, which in turn must contain a `max`
-aggregation on the time field. This requirement ensures that the aggregated data
-is a time series and the timestamp of each bucket is the time of the last record
-in the bucket.
+There are some limitations to using aggregations in {dfeeds}.
 
-IMPORTANT: The name of the aggregation and the name of the field that the agg 
-operates on need to match, otherwise the aggregation doesn't work. For example, 
-if you use a `max` aggregation on a time field called `responsetime`, the name 
+Your aggregation must include a `date_histogram` aggregation or a top level `composite` aggregation,
+which in turn must contain a `max` aggregation on the time field.
+This requirement ensures that the aggregated data is a time series and the timestamp
+of each bucket is the time of the last record in the bucket.
+
+IMPORTANT: The name of the aggregation and the name of the field that the agg
+operates on need to match, otherwise the aggregation doesn't work. For example,
+if you use a `max` aggregation on a time field called `responsetime`, the name
 of the aggregation must be also `responsetime`.
 
-You must also consider the interval of the date histogram aggregation carefully.
-The bucket span of your {anomaly-job} must be divisible by the value of the
-`calendar_interval` or `fixed_interval` in your aggregation (with no remainder).
-If you specify a `frequency` for your {dfeed}, it must also be divisible by this
-interval. {anomaly-jobs-cap} cannot use date histograms with an interval 
-measured in months because the length of the month is not fixed. {dfeeds-cap} 
-tolerate weeks or smaller units.
+You must consider the interval of the `date_histogram` or `composite`
+aggregation carefully. The bucket span of your {anomaly-job} must be divisible
+by the value of the `calendar_interval` or `fixed_interval` in your aggregation
+(with no remainder). If you specify a `frequency` for your {dfeed},
+it must also be divisible by this interval. {anomaly-jobs-cap} cannot use
+`date_histogram` or `composite` aggs with an interval measured in months
+because the length of the month is not fixed. {dfeeds-cap} tolerate weeks or smaller units.
```

Comment:
The use of "tolerate" and the use of "jobs" in one sentence and "datafeeds" in the other seemed unnecessarily confusing. I think it's clearer to say something like this:
```suggestion
because the length of the month is not fixed; they can use weeks or smaller units.
```